big_scape/output/legacy_output.py

```diff
@@ -841,6 +843,9 @@ def generate_bs_families_alignment(
         # collects records within this GCF
         family_records = [records[bgc_num] for bgc_num in family_members]
         family_member_db_ids = [rec._db_id for rec in family_records]
+        if family_db_id not in family_member_db_ids:
+            continue
+        # TODO: get rid of this once bug in fam assignment fixed
```

Comment:
this should be in dev now, let me know if it's not

Reply:
I dont think so? maybe i'm missing it somehow